common/artifacts/000_wc_artifacts.txt

```diff
@@ -818,6 +1133,36 @@ chalice_of_trollbanes = {
 
 	picture = "GFX_inv_misc_trophy_argent"
 }
+legion_doomhorn = {
+	quality = 3
+	flags = { misc rare }
+
+	active = {
+		is_adult = yes
+		OR = {
+			is_warlock_class_trigger = yes
+			religion_group = fel_group	
+		}
+	}
```

Comment:
I would use here `use_dark_artifacts_trigger = yes`. Strange that only warlocks and Fel followers can use it while in the strategy the artifact is available to everyone.

Reply:
ok, done